openlibrary/templates/site/footer.html

```diff
@@ -22,8 +22,7 @@
       if ("serviceWorker" in navigator) {
         window.addEventListener('load', () => {
           navigator.serviceWorker.register("/sw.js").then(registration => {
-            console.log('ServiceWorker registration successful with scope: ', registration.scope);
-            console.log(registration);
+
           }).catch(error => {
             console.log("ServiceWorker registration failed");
             console.log(error);
```

Comment:
Can you also remove the `console.log(error)` in the `catch block?  Printing the error itself may expose too much information to patrons.

The "ServiceWorker registration failed" log statement doesn't need to be removed now, as it may be helpful to folks who are working on the PWA/TWA project.